src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
```

Comment:
Use arithmetic operators with assignment operator `=` to add to the existing value. This is a code style issue that makes the code shorter and more readable.

BAD EXAMPLE:
 return this.result + num;

GOOD EXAMPLE:
 this.result += num;

Please update the `add` method to use `+=` instead of `+`.